sources/ec.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+      "url": "http://www.aqvisor.net:9090/openaq/dataset",
```

Comment:
I might change this to just be `http://www.aqvisor.net:9090/openaq/` so that you can use the `location` and `dataset` endpoints in the adapter